app/views/cases/_new.html.slim

```diff
@@ -2,7 +2,7 @@
   = t('.heading')
 
 - content_for :sub_heading
-  = t4c(kase, '', 'sub_heading', case_type: kase.type_abbreviation)
+  = get_offender_sar_heading(@offender_sar_type, @case)
```

Comment:
This needs to be more generic so it works for any case type. At the moment it's going to break the new case page for FOIs and SARs.

To start you should create a method that takes `kase` as a param and then have it output the exact same thing it does now.

Then you need to work out how to determine when the new case is a rejected offender sar based on information in that kase variable.

Reply:
It is also possible that you can made changes to the `TranslateForCase` class (which is what the t4c method eventually calls to account for a rejected offender sar there.

Reply:
I am thinking of adding a flag to set on the case if it's a rejected offender sar then the t4c can check and output the rejected heading if its an offender sar & rejected.
What do you think?

Reply:
That seems like a reasonable idea.